src/kOS/SharedObjects.cs

```diff
@@ -12,15 +12,27 @@ public class SharedObjects : Safe.SafeSharedObjects
         public ProcessorManager ProcessorMgr { get; set; }
         public Part KSPPart { get; set; }
         public TermWindow Window { get; set; }
+        public List<KOSManagedWindow> ManagedWindows { get; }
```

Comment:
I'm compiling a list of all these "should be addressed, but shouldn't prevent the GUI from being merged" kinds of issues and I added them to the issues list.  This is one of them, here: https://github.com/KSP-KOS/KOS/issues/1952